app/src/main/java/com/github/bsideup/liiklus/config/GatewayConfiguration.java

```diff
@@ -13,11 +14,12 @@
 import java.util.Map;
 import java.util.stream.Collectors;
 
-public class LayersConfiguration implements ApplicationContextInitializer<GenericApplicationContext> {
+public class GatewayConfiguration implements ApplicationContextInitializer<GenericApplicationContext> {
```

Comment:
so this is the only configuration left in main?

Reply:
yup :)